lua/mason-null-ls/mappings/language.lua

```diff
@@ -0,0 +1,25 @@
+local registry = require "mason-registry"
+
+local M = {}
+
+---Returns a map of language (lowercased) to one or more corresponding Mason package names.
+---@return table<string, string[]>
+function M.get_language_map()
+    if not registry.get_all_package_specs then
+        return {}
+    end
```

Comment:
This was a new function just added to mason, it returns only the underlying package metadata tables instead of instantiating Package instances. About a 70% perf boost.